### ISSUE TITLE
Promote "7guis" tag

### ISSUE THREAD

**Issue by tbreuss** [7guis/7guis#25]
~~Several  repos~~ 35 public repositories on GitHub are using the `7guis` tag already. And it seems, that the tag name really is unique, and kind of "referencing" to this repo.

See repositories matching the `7guis` topic: https://github.com/topics/7guis

We should promote this tag somewhere on the website as a simple solution to contribute to 7Guis.

Additionally, we could add a link to the above Github topic, too.

**Comment by tbreuss** [comment]
I just added the `7guis` tag on this repo.

**Comment by dumblob** [comment]
Yes, nice idea! I will keep this in mind and promote the tag elsewhere where it makes sense.
